src/core/checkedint.d

```diff
@@ -682,10 +682,27 @@ unittest
 
 /// ditto
 pragma(inline, true)
+ulong mulu(uint x, ulong y, ref bool overflow)
+{
+    ulong r = x * y;
+    if (y >> 32 && x && (r / x) != y)
```

Comment:
@wilzbach I know... we should get better at this either by count or by supporting columns.